vignette/Validation.Rmd

```diff
@@ -0,0 +1,155 @@
+---
+title: "Comparison"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Comparison}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>", 
+  fig.path = "man/figures/README-",
+  fig.width = 9, 
+  fig.height = 9
+)
+```
+
+```{r setup}
+library(hypatia)
+library(ggplot2)
+library(tidyverse)
+```
+
+```{r hypatia}
+
+# Create our population
+countryname = "Antigua and Barbuda"
+
+pop <- hypatia:::get_population(countryname)
+
+# Scale it for speed
+pop$n <- round(pop$n/10)
+
+# Create our equivalent parameters
+time_period <- 200
+
+parameters <- get_parameters(
```

Comment:
unlike the other functions this one does not have `hypatia::` - either all calls into the package should be namespace qualified or all (I'd prefer all)